reportportal_client/service.py

```diff
@@ -189,7 +189,13 @@ def start_test_item(self, name, start_time, item_type, description=None,
             }
         """
         if parameters is not None:
-            parameters = [{"key": key, "value": str(value)}
+            def _str(value):
+                if isinstance(value, unicode):
+                    return value.encode("utf-8")
```

Comment:
This is wrong to encode value to `utf-8` due to fact that `body` passed to json marshal and result encoded to `utf-8` by request lib: https://github.com/psf/requests/blob/d2590ee46c0641958b6d4792a206bd5171cb247d/requests/models.py#L469

The quick fix is just covert to unicode and in case of `NameError` to `str` for Py3. E.g.
```python
def _str(value):
    try:
         return unicode(value)
    except NameError:
        return str(value)
```

Reply:
So, `json.dumps` will raise `TypeError` when we have bytes values in case of python3, `six.text_type(value)` looks cool, thank you!